app/models/user.rb

```diff
@@ -1,2 +1,16 @@
 class User < ApplicationRecord
+  has_many :posts, dependent: :destroy
```

Comment:
- [ ] When trying to get the posts for a particular user using **Active Record** (`first_user.posts`) I got an error (see image). This it's because the foreign_key option is not defined in the User model. Kindly add **author_id** as the `foreign_key` for the post

Example: `has_many :posts, foreign_key: 'author_id', dependent: :destroy`
![Screenshot from 2024-01-11 08-41-27](https://github.com/George7h/blog/assets/91142432/aa533cd4-5cc1-4568-a922-66443362ee49)